mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -4376,21 +4380,13 @@ fn test_block_header_utxo_loop() {
         "max_stored_block_headers": 15
     }));
 
-    let loop_fut = async move {
-        block_header_utxo_loop(
-            arc.downgrade(),
-            UtxoStandardCoin::from,
-            loop_handle,
-            unsafe { CURRENT_BLOCK_COUNT },
-            spv_conf.unwrap(),
-        )
-        .await
-    };
+    let weak_client = Arc::downgrade(&client.0);
+    let loop_fut = async move { block_header_utxo_loop(weak_client, loop_handle, spv_conf.unwrap()).await };
 
     let test_fut = async move {
         *expected_steps.lock().unwrap() = vec![(2, 5), (6, 9), (10, 13), (14, 14)];
         unsafe { CURRENT_BLOCK_COUNT = 14 }
-        Timer::sleep(3.).await;
+        Timer::sleep(4.).await;
```

Comment:
please help me reverse this back to `3.`

Reply:
Done